_projects/Lossy_checkpoint_restart.md

```diff
@@ -0,0 +1,62 @@
+
+Few studies have been done on checkpointing and restart from lossy compressed states [Sas15]. These studies are limited in scope by studying only one application and one type of lossy compressor: NICAM with a lossy compressor based on wavelet and quantization [Sas15] and ChaNGa with the fpzip lossy compressor [Ni14]. They already reveal two interesting points/First, checkpoints are composed of different variables that present different sensitivity to lossy compression. The correctness of the execution after restart depends on how each variable is compressed. In the cosmology simulation (ChaNGa), lossy compression of particle positions lead the execution to hang for a high compression level, while this does not happen when compressing other variable at the same level. Second, for the study of NICAM, the authors consider that an error of 1\% on the final result is acceptable when restarting from lossy checkpoints. The rationale is that this magnitude of error is similar to those of sensor errors and model errors, while the compression factor exceeds 5. 
+
+In contrary to previous research that concentrated on few applications, we focus on simple problems used by many applications and try to understand how they behave. We are exploring diffusion and advection problems. The diffusion problem simulates the heat diffusion on a one-dimensional rod. The advection problem simulates a sine wave advecting to the right with periodic boundaries. 
+
+## Results for 2015/2016
+
+From our observations of restart from lossy checkpoints in dynamic simulations, we have observed that diffusion and advection problems are reacting differently. We are currently characterizing and formalizing this difference. We suspect that lossyness may be increased as the execution progresses: restarting from a lossy checkpoint near the beginning of the execution seems to affect much more the end results compared with restarting from a lossy checkpoint near the end of the execution. Moreover, we note that in a given variable field, some regions of the field are more chaotic than others. Some applications may not tolerate lossy compression on the entire field. But we suspect that they may tolerate lossy compression in the nonchaotic regions of the variable field. 
+
+## Visits and meetings
+
+Franck Cappello visits UIUC almost every week. We have a 30 minutes-1hour meeting almost each time.
+
+## Impact and publications
+
+{% bibliography --cited --file jlesc.bib %}
+
+
+## Person-Month efforts in 2015/2016
+
+{:.person-months-table.table.table-bordered.table-hover.table-sm}
+| {% person cappello_f %} | 1 PM |
+| {% person olson_l %} | 1 PM |
+| {% person calhoun_j %} | 3 PM |
+
+## Future plans
+
+Jon Calhoun will spend the summer at ANL with Franck Cappello for an internship.
+A paper is in preparation for a submission in a conference with a deadline in the 1/2 half of 2016.
+
+## References
+
```

Comment:
Please use the format specified in the template (see also the other projects, e.g. https://github.com/JLESC/jlesc.github.io/blob/source/_projects/folding_hmem.md)